views-core/src/test/groovy/io/micronaut/views/ViewUtilsSpec.groovy

```diff
@@ -0,0 +1,19 @@
+package io.micronaut.views
+
+import spock.lang.Specification
+
+class ViewUtilsSpec extends Specification {
```

Comment:
please merge `master` against this branch and keep the version of this file present in `master`